src/components/Tooltip/Tooltip.tsx

```diff
@@ -1,69 +1,80 @@
-
-export const Tooltip: FC<TooltipProps> = ({ tooltip }) => {
-    const [tooltipTriggerElement, setTooltipTriggerElement] = useState<HTMLElement | null>(null);
-    const [tooltipElement, setTooltipElement] = useState<HTMLDivElement | null>(null);
-    const state = useTooltipTriggerState();
-    const { triggerProps, tooltipProps } = useTooltipTrigger({}, state, useRef(tooltipTriggerElement));
-    const { tooltipProps: tooltipAriaProps } = useTooltip(tooltipProps, state);
-    const { isOpen } = state;
-    const { isFocusVisible } = useFocusVisible();
-    const { styles, attributes } = usePopper(tooltipTriggerElement, tooltipElement, {
-        placement: "auto-end",
-        modifiers: [
-            {
-                name: "offset",
-                options: {
-                    offset: [TOOLTIP_SKIDDING, TOOLTIP_DISTANCE],
-                },
-            },
-        ],
-    });
-
+export const Tooltip: FC<TooltipProps> = ({
+    tooltip,
+    tooltipIcon,
+    heading,
+    headingIcon,
+    linkUrl,
+    linkLabel,
+    brightHeader,
+    buttons,
+}) => {
     return (
         <>
-            <button
-                {...triggerProps}
-                data-test-id="tooltip-icon"
-                ref={setTooltipTriggerElement}
-                onMouseEnter={() => state.open()}
-                onMouseLeave={() => state.close(true)}
-                className={merge([
-                    "tw-inline-flex tw-items-center tw-justify-center tw-text-black-60 hover:tw-text-black dark:tw-text-black-40 dark:hover:white tw-cursor-default tw-outline-none tw-rounded-full",
-                    isOpen && isFocusVisible && FOCUS_STYLE,
-                ])}
-            >
-                <IconQuestion size={IconSize.Size16} />
-            </button>
-            {isOpen && (
-                <div
-                    {...tooltipAriaProps}
-                    data-test-id="tooltip"
-                    ref={setTooltipElement}
-                    style={styles.popper}
-                    {...attributes.popper}
-                    className="tw-p-4 tw-border tw-border-black-10 tw-bg-white tw-rounded-md tw-shadow-mid dark:tw-bg-black-90 dark:tw-text-white tw-z-20"
-                >
-                    {tooltip}
+            <div className="tw-inline-block tw-max-w-[200px] tw-bg-black-100 dark:tw-bg-white tw-rounded-md tw-shadow-mid tw-text-white dark:tw-text-black-100 tw-z-20">
+                {brightHeader && <BrightHeader headerStyle={brightHeader} />}
+                <div data-test-id="tooltip" className="tw-p-4">
+                    {heading && headingIcon && (
+                        <h4 className="tw-flex tw-text-m tw-font-bold tw-mb-1">
+                            <span className="tw-mr-1.5">
+                                {headingIcon && cloneElement(headingIcon, { size: IconSize.Size20 })}
+                            </span>
+                            {heading}
+                        </h4>
+                    )}
```

Comment:
```suggestion
                    {heading && (
                        <h4 className="tw-flex tw-text-m tw-font-bold tw-mb-1">
                            <span className="tw-mr-1.5">
                                {headingIcon && cloneElement(headingIcon, { size: IconSize.Size20 })}
                            </span>
                            {heading}
                        </h4>
                    )}
```

This shouldn't happen you may want to have the heading without any icon (this behavior already have been asked to Simone for previous components)